packages/strapi-design-system/src/SimpleNav/SimpleNavItem.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { BaseLink } from '../BaseLink';
+import { Box } from '../Box';
+import { Flex } from '../Flex';
+import { Typography } from '../Typography';
+import { buttonFocusStyle } from '../themes/utils';
+
+const SimpleNavItemWrapper = styled(Box)`
+  text-decoration: none;
+  background: ${({ background, theme }) => theme.colors[background] || background};
+  color: ${({ textColor, theme, withError }) => theme.colors[withError ? 'danger600' : textColor]};
+
+  &.active,
+  &:active,
+  &[aria-current='page'] {
+    background: ${({ theme, withError }) => theme.colors[withError ? 'danger100' : 'primary100']};
+    color: ${({ theme, withError }) => theme.colors[withError ? 'danger600' : 'primary700']};
+  }
+
+  &:not(.active):not(:active):not([aria-current='page']) {
+    &:hover {
+      background: ${({ theme, withError }) => theme.colors[withError ? 'danger100' : 'neutral150']};
+      color: ${({ theme, withError }) => theme.colors[withError ? 'danger600' : 'neutral800']};
+    }
+  }
+
+  svg,
+  svg * {
+    fill: currentColor;
+  }
+
+  ${buttonFocusStyle}
+`;
+
+const SimpleNavItemIconWrapper = (props) => <Flex as="span" alignItems="center" aria-hidden="true" {...props} />;
+
+export const SimpleNavItem = ({
+  children,
+  className,
+  endIcon,
+  gap,
+  isActive,
+  startIcon,
+  textVariant,
+  withError,
+  ...rest
+}) => {
+  return (
+    <SimpleNavItemWrapper className={`${className}${!isActive ? '' : ' active'}`} withError={withError} {...rest}>
```

Comment:
I would just pass `isActive` as a prop because this string concatenation can get messy without a lib like `clsx`.